.github/workflows/actions.yml

```diff
@@ -153,15 +137,7 @@ jobs:
         if: env.USE_SYSLIBS == 'true'
         run: sudo apt-get install --yes libboost-thread-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libboost-test-dev libboost-program-options-dev libyaml-cpp-dev
       - name: install qt from apt
-        if: '!matrix.qt-version'
         run: sudo apt-get install qt5-default qt5-qmake qttools5-dev qttools5-dev-tools qtdeclarative5-dev libqt5svg5-dev libqt5websockets5-dev qtwebengine5-dev
-      - name: install qt using aqtinstall
```

Comment:
Thanks!
Do we want to remove the functionality to install arbitrary Qt version? Maybe it would be useful in the future in we e.g. start offering Linux app package (Appimage etc)?
I have slight preference to leave it in (along with the caching step) and if you feel strongly if should be removed I'd be fine with that.

Reply:
Yes, I feel quite strongly about it -- leaving dead code around is confusing and unnecessary when we have version control. If there is not an immediate reason to keep it (i.e. it would make work we are already planning to do easier), we should remove it.

Reply:
Sounds good!